src/main/java/page/GeneralPage.java

```diff
@@ -2,26 +2,71 @@
 
 import common.Constant;
 import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+import java.util.List;
+
 public class GeneralPage {
-    private WebDriver driver;
-    private By tabLogin = By.xpath("//a[contains(@href,'Login')]");
-    private By tabRegister = By.xpath("//a[contains(@href,'Register')]");
+    private String Login = "Login";
+    private String Register = "Register";
+    private String BookTicket = "BookTicketPage";
+    private String Logout = "Logout";
+    private String MyTicket = "ManageTicket";
+    private String ChangePassWord = "ChangePassword";
```

Comment:
please use them (Login/Register/...) in the enum or constant

Reply:
I will create a enum class to contain it